### DISCUSSION TITLE
The ability to exclude a directory from watching in tailwind.config.js would be very helpful.

### DISCUSSION THREAD

**Discussion by zbandhan** [tailwindlabs/tailwindcss#9515] [Ideas]
Tailwind has an option to include directories to purge CSS classes from. But an exclude option could come very handy in some situations. Mostly we need to exclude node_modules, assets and others. Is it possible to add this feature in Tailwindcss?

![image](https://user-images.githubusercontent.com/10957285/194793015-f0a88252-857f-4a39-855a-58012a480f77.png)

**Comment by wongjn** [comment]
I believe this feature is already available in Tailwind. You should be able to add negating patterns to add exclusion globs:
```js
module.exports = {
  content: [
    './src/**/*.js',
    '!./src/foo',
  ],
};
```

**Reply by k-utsumi** [reply]
It's working fine.

```js
module.exports = {
  content: [
    "./src/**/*.{html,js}",
    "!./src/**/node_modules/**/*.{html,js}" // <- Add
  ],
  ...
}
```

Build speed has been improved.
<table role="table">
<thead>
<th>Before
<th>After
<tbody>
<td>

```sh
Rebuilding...

Done in 18844ms.
```
<td>

```sh
Rebuilding...

Done in 151ms..
```
</table>
